run_scripts/1_coda_get.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 3 ]]; then
+    echo "Usage: ./1_coda_get.sh <coda-auth-file> <coda-v2-root> <data-root>"
+    echo "Downloads coded messages datasets from Coda to '<data-root>/Coded Coda Files'"
+    exit
+fi
+
+AUTH=$1
+CODA_V2_ROOT=$2
+DATA_ROOT=$3
+
+./checkout_coda_v2.sh "$CODA_V2_ROOT"
+
+cd "$CODA_V2_ROOT/data_tools"
+git checkout "9a9a8e708e3f20f37848a6b02f79bcee43e5be3b"  # (master which supports segmenting)
+
+mkdir -p "$DATA_ROOT/Coded Coda Files"
+
+DATASETS=(
+    "UNICEF_COVID19_SOM_s01e01"
+    "UNICEF_COVID19_SOM_s01e02"
+    "UNICEF_COVID19_SOM_s01e03"
+    "UNICEF_COVID19_SOM_s01e04"
+
+    "UNICEF_COVID19_SOM_location"
```

Comment:
If this is sharing data with imaqal, does it make sense to share the demog dataset and use IMAQAL_location instead?

Reply:
so we have decided to start afresh for this project and not use Imaqal data.

Reply:
Great, that makes things easy for us :)